### DISCUSSION TITLE
Anyone created dropdowns / select components in toolbars, how did you manage the cursor state?

### DISCUSSION THREAD

**Discussion by deep-c** [ueberdosis/tiptap#1243] [General Questions] [answered]
Hi,

Just wondering if anyone had used https://headlessui.dev/react/listbox or another custom select component. How did you manage the cursor state? Did you do some work around toggling read-only state and keeping track of the last position or was there an easier method?

Keen to know how to manage this in this library.

**Comment by philippkuehn** [comment, marked as answer]
Just use the `focus()` command to restore your selection.

**Reply by deep-c** [reply]
doh... thanks for that. seems like using headlessui components have some weird focus accessibility behavior so i rolled my own using the command which worked.
